api-reference/10 UI Components/dxVectorMap/1 Configuration/layers/paletteIndex.md

```diff
@@ -4,7 +4,7 @@ type: Number
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+The position of a color in the [palette[]](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/layers/#palette) array. Cannot exceed the value of the [paletteSize](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/layers/#paletteSize) property. 
```

Comment:
We need to elaborate on the purpose and usage of this property. Now, the only thing that's clear is that it is somehow used to color the map. What elements does this property color? How to decide which value I should set it to? What are common use-cases? Code examples or demos?

Reply:
This property is just an addition for the main property - palette[]. This is also the answer to most of your questions - it is an index used to iterate through an array of colors (predefined or custom palette).